pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -518,6 +518,11 @@ func (configgen *ConfigGeneratorImpl) buildSidecarInboundListeners(
 				bindToPort: false,
 			}
 
+			if sidecarScope.Config != nil {
+				rule := sidecarScope.Config.Spec.(*networking.Sidecar)
```

Comment:
nit: make this conversion in front of  if

Reply:
we can't because sidecarScope could be a default one without the Config right?